### PR TITLE
Homework 9:

### PR DESCRIPTION
**Pull request by chatieb**
Part 1
Make a program which has 2 conversion modes:
From decimal to binary
From binary to decimal

Part 2
Use your WASD (both capital and non capital) keys to print arrows
W- ↥
A- ↤
S- ↧
D- ↦

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -1,12 +1,209 @@
...
+
+        private const char LeftKey = 'A';
+        private const char UpwardKey = 'W';
+        private const char RightKey = 'D';
+        private const char DownKey = 'S';
+
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Part 1 - Convert with 2 conversion modes: From decimal to binary, From binary to decimal.
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: This could be moved as XML docs of method.

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -1,12 +1,209 @@
...
+            ConvertNumberSystem();
+
+            //Part 2 - Use your WASD (both capital and non capital) keys to print arrows.
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: This could be moved as XML docs of method.

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -1,12 +1,209 @@
...
+            ConvertLettersToArrows();
+        }
+
+        private static void ConvertNumberSystem()
+        {
+            var mode = GetConversionMode();
+            var number = GetNumber(mode);
+            var convertedNumber = (mode == 1) ? ConvertDecimalToBinary(int.Parse(number)) : ConvertBinaryToDecimal(number).ToString(); ;
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Program.cs]: 1 serves a bit as a magic number here. We will tackle this later on but you could have resolved that by using another constant variable. This is a minor issue though. :)

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -1,12 +1,209 @@
...
+            var mode = 0;
+            do
+            {
+                Console.WriteLine("Choose 1 or 2:");
+                isValidMode = int.TryParse(Console.ReadLine(), out mode);
+
+                if (!isValidMode && (mode != 1 && mode != 2))
+                {
+                    isValidMode = false;
+                }
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Program.cs]: Here we can clearly see what happens if you choose to ever use other methods/values. That will need a lot of changes. That is why I did point out using constants (for now) could be helpful.

**Pull request closed without merging**